joblib/memory.py

```diff
@@ -685,6 +685,11 @@ def check_call_in_cache(self, *args, **kwargs):
         func_id, args_id = self._get_output_identifiers(*args, **kwargs)
         return self.store_backend.contains_item((func_id, args_id))
 
+    def invalidate_cache(self, *args, **kwargs):
+        """Removes this call from the cache"""
+        identifier = self._get_output_identifiers(*args, **kwargs)
```

Comment:
Note this private API has changed. Whoever is taking this PR over will need to use: `identifier = (self.func_id, self._get_args_id(*args, **kwargs))`